lib/middleware/check-progress.js

```diff
@@ -18,7 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
 
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = controller._getForkTarget(req, res);
```

Comment:
True that. Why don't we go with both? provide a public method on form-controller, and then call that directly in this middleware to protect against this method being extended in child controllers?